isis/version

```diff
@@ -1,3 +0,0 @@
-3.9.0        # Public version number
```

Comment:
Can this be removed without breaking `Environment::isisVersion()`?

Reply:
ISISROOT will be set to the binary directory, which will contain the version file. The configure_file call writes it out to the same location as it was copied to before.

Reply:
So, the removal of this file will have no impact, because the code doesn't point back into the source tree. It points into the external build now.

Reply:
You're right, thank you. I thought it was writing it back out to something that was still called `version.in`, rather than `version`.

Reply:
yeah, the `*.in` format is a standard file naming convention for CMake rendered files.